src/components/data-grid/DataGridActions.tsx

```diff
@@ -66,8 +68,28 @@ export const DataGridActions = ({
             optionProps: () => ({ [DATA_TEST_ID]: EXPORT_OPTION_TEST_ID }),
           }}
         >
-          <TertiaryButton disabled={!rowsSelected} testId={EXPORT_BUTTON_TEST_ID}>
-            <Download size={scale600} color={rowsSelected ? primaryA : contentStateDisabled} />
+          <TertiaryButton
+            overrides={{
```

Comment:
I think this is fine for now. But I wanna avoid overrides as much as possible. so later we're gonna just update the styling of the general `Button` component so that we can just use it here without overrides